www/src/pages/community/people.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react"
+import { graphql } from "gatsby"
+import CommunityView from "../../views/community"
+
+class PeoplePage extends Component {
+  render() {
+    const { location, data } = this.props
+    return <CommunityView data={data} location={location} title={`People`} />
+  }
+}
+
+export default PeoplePage
+
+export const pageQuery = graphql`
+  query {
+    allCreatorsYaml(filter: { type: { eq: "individual" } }) {
```

Comment:
It's hard to tell at a glance, but these are all pretty similar queries right? You may look into a fragment (generally co-located with the component that they're injected into, e.g. CommunityView) of the minimum subset of fields, and then use that fragment appropriately.

[Here's an example](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/blog-post-preview-item.js#L126)

For this, it'd look something like

```
import { graphql } from 'gatsby'

// the component

export const communityFragment = graphql`
  fragment CommunityFragment on CreatorsYaml {
    name
    description
    # etc
  }
`;
```

and then to use it, it looks kinda like object rest/spread, e.g.

```js
import { graphql } from 'gatsby';

// the component

export const pageQuery = graphql`
  query {
    allCreatorsYaml(filter: { type: { eq: "individual } }) {
      ...CommunityFragment
      for_hire
      portfolio
    }
  }
`;
```

hope this helps!

Reply:
Thank you, great suggestion! 👍